src/styles/variables/variables.scss

```diff
@@ -0,0 +1 @@
+$transition: transform 300ms, color 300ms;
```

Comment:
minor in real life such variables are rear cases. in your case it was better to move 300ms to variable only. As nav links do not change their transform property on hover event.